### PR TITLE
Child theme

### PR DESCRIPTION
**Pull request by spence-man**

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -2,8 +2,8 @@
 # This should be filled out and placed in /chap-press/config/database.yml
 # This file is *only* used to generate wp-config.php
 staging:
-  host: 'TBA' 
+  host: 'localhost' 
   database: 'chappress_dev' 
   username: 'chappress'
-  password: 'TBA'
+  password: 'chappress'
```
Comment (tatwell) [inline config/database.yml]: Can we remove this file? We have `database.yml-dist`.

File: config/deploy.rb [hunk 2]
```diff
@@ -21,7 +21,7 @@
 set :use_sudo, false
 set :pty, true
 set :ssh_options, forward_agent: true
-set :keep_releases, 5
+set :keep_releases, 15
```
Comment (tatwell) [inline config/deploy.rb]: `5` should be plenty here. Any reason for `15`?

File: config/deploy.rb [hunk 3]
```diff
@@ -46,6 +46,7 @@
   task :wp_permissions do
     on roles(:app) do
       execute :chmod, "644 #{release_path}/public/wp-config.php"
+      execute :chmod, "-R 755 #{release_path}/content/uploads"
```
Comment (tatwell) [inline config/deploy.rb]: Aren't files usually `644`? This appears to make everything `755`. 

Do we have a reference for this? I'd like to see a code comment that assures us we based these settings on a reliable source.

**Review by tatwell — COMMENTED** [review]
Looks great. Still few questions about config stuff.

File: config/deploy.rb (hunk 2 above)
Comment (spence-man) [reply config/deploy.rb]: Oh yes, i'll change that back to 5. I was just testing and troubleshooting deployment

File: content/themes/chappress/main.scss [hunk 4]
```diff
@@ -62,7 +62,7 @@ a {
   line-height: 20px;
 }
 
-.custom-logo-link {
-  background-image: url("http://localhost:8222/wp-content/themes/chappress/assets/images/cu-logo.png");
+.chapman-logo-link {
+  background-image: url("https://chappress-staging.chapman.edu/wp-content/themes/chappress/assets/images/cu-logo.png");
```
Comment (tatwell) [inline content/themes/chappress/main.scss]: A relative url works here, yes?

    background-image: url("/wp-content/themes/chappress/assets/images/cu-logo.png");

File: content/themes/chappress/style.css [hunk 5]
```diff
@@ -122,8 +122,8 @@ a {
   font-size: 1rem;
   line-height: 20px; }
 
-.custom-logo-link {
-  background-image: url("http://localhost:8222/wp-content/themes/chappress/assets/images/cu-logo.png");
+.chapman-logo-link {
+  background-image: url("https://chappress-staging.chapman.edu/wp-content/themes/chappress/assets/images/cu-logo.png");
```
Comment (tatwell) [inline content/themes/chappress/style.css]: See last comment regarding relative url.

File: content/themes/chappress/main.scss (hunk 4 above)
Comment (spence-man) [reply content/themes/chappress/main.scss]: Removed that css class, it was left accidentally and wasn't loading.

File: content/themes/chappress/style.css [hunk 6]
```diff
@@ -122,8 +122,4 @@ a {
   font-size: 1rem;
   line-height: 20px; }
 
-.chapman-logo-link {
-  background-image: url("https://chappress-staging.chapman.edu/wp-content/themes/chappress/assets/images/cu-logo.png");
-  background-size: 100%; }
-
```
Comment (tatwell) [inline content/themes/chappress/style.css]: Are these deletions not needed? Logo is still present?

Comment (spence-man) [reply content/themes/chappress/style.css]: The login logo is active and is fine. I was trying to customize the home page logo but having issues with the customizing the hook into that.

**Review by tatwell — APPROVED** [review]
I've kinda lost track of this. If all the issues I've noted have been resolved, I approve merging.

**Pull request merged**